micrometer-core/src/main/java/io/micrometer/core/instrument/util/DoubleFormat.java

```diff
@@ -69,8 +69,19 @@ public static String decimalOrNan(double d) {
      * @param d Number to format.
      * @return A stringified version of the number that only uses a decimal representation if the number is not
      * whole.
+     * @deprecated since 1.0.6 in favour of {@link #wholeOrDecimal(double)}
```

Comment:
```suggestion
     * @deprecated since 1.0.11 in favour of {@link #wholeOrDecimal(double)}
```

Reply:
@shakuzen I applied the suggested change in 81cab5dacf4e10cc6c89e9ad2bf4343caf02ae7d.